app/lib/send-discord.ts

```diff
@@ -52,7 +52,7 @@ export async function sendDiscord(
 
 function createWebhookPayload(event: AnyWebhookEvent): DiscordWebhookPayload {
 	const appName = "Todo List";
-	const baseUrl = process.env.BASE_URL || process.env.VERCEL_URL;
+	const baseUrl = process.env.BASE_URL || `https://${process.env.VERCEL_URL}`;
```

Comment:
Not everyone may be using vercel... So it may not be set at all.

Meaning for such environments, you'll get `https://undefined`